pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -287,6 +287,19 @@ const (
 	// LabelMonitoring is a constant for a label for monitoring stack configurations
 	LabelMonitoring = "monitoring"
 
+	// LabelExtensionTypePrefix is used to prefix extension label for extension types.
+	LabelExtensionTypePrefix = "extensions.extensions.gardener.cloud/"
```

Comment:
Nit: Technically and for symmetry reasons, it should be as follows, even if it looks odd :)

```suggestion
	LabelExtensionExtensionTypePrefix = "extensions.extensions.gardener.cloud/"
```

Reply:
Btw, why does it contain "Type" while the other constants names to not contain "Type" in their names? Please harmonize this.